index.html

```diff
@@ -352,6 +365,94 @@
+      </p>
+
+      <dl title='interface GamepadPose' class='idl'>
+        <dt>readonly attribute Float32Array? position</dt>
+        <dd>
+          Position of the gamepad as a 3D vector. Position is given in meters
+          from an origin point, which is determined by the gamepad hardware, and
+          may be the position of the gamepad when first polled if no other
+          reference point is available. The coordinate system uses these axis
+          definitions, assuming the user is holding the gamepad in the forward
+          orientation:
+
+          <ul>
+            <li>Positive X is to the user's right.</li>
+            <li>Positive Y is up.</li>
+            <li>Positive Z is behind the user.</li>
+          </ul>
+
+          MUST be null if the gamepad is incapable of providing positional data.
+          When not null MUST be a three-element array.
+        </dd>
+
+        <dt>readonly attribute Float32Array? linearVelocity</dt>
+        <dd>
+          Linear velocity of the gamepad in meters per second. MUST be null if
+          the sensor is incapable of providing linear velocity. When not null
+          MUST be a three-element array.
+        </dd>
+
+        <dt>readonly attribute Float32Array? linearAcceleration</dt>
+        <dd>
+          Linear acceleration of the gamepad in meters per second. MUST be null
+          if the sensor is incapable of providing linear acceleration. When not
+          null MUST be a three-element array.
+        </dd>
+
+        <dt>readonly attribute Float32Array? orientation</dt>
+        <dd>
+          Orientation of the gamepad as a quaternion. An orientation of
+          [0, 0, 0, 1] is considered to be "forward". The forward direction is
+          determined by the gamepad hardware, and may be the orientation of the
+          hardware when it was first polled if no other reference orientation is
+          available. MUST be null if the sensor is incapable of providing
+          orientation data. When not null MUST be a four-element array.
+        </dd>
+
+        <dt>readonly attribute Float32Array? angularVelocity</dt>
+        <dd>
+          Angular velocity of the gamepad in meters per second. MUST be null if
+          the sensor is incapable of providing angular velocity. When not null
+          MUST be a three-element array.
+        </dd>
+
+        <dt>readonly attribute Float32Array? angularAcceleration</dt>
+        <dd>
+          Angular acceleration of the gamepad in meters per second. MUST be null
+          if the sensor is incapable of providing angular acceleration. When not
+          null MUST be a three-element array.
+        </dd>
+      </dl>
+    </section>
+
+    <section>
+      <h2><dfn>GamepadCapabilities</dfn> Interface</h2>
+      <p>
+        This interface defines the gamepad's hardware capabilities.
+      </p>
+
+      <dl title='interface GamepadCapabilities' class='idl'>
+        <dt>readonly attribute boolean hasPosition</dt>
```

Comment:
I'm comfortable with `Gamepad#pose`, but this one not so much. can you explain other instances that are not specific to VR when we need capabilities info. that cannot be discerned from the presence of buttons and axes and their values?